src/api/flaskr/service/user/google_oauth.py

```
@@ -0,0 +1,167 @@
+# Google OAuth service
+# author: claude
+
+import hashlib
```

Ruff (0.12.2)

`hashlib` imported but unused

Remove unused import: `hashlib`

(F401)